physionet-django/console/views.py

```diff
@@ -274,6 +275,17 @@ def submission_info(request, project_slug):
         LOGGER.info("The editor for the project {0} has been reassigned from "
                     "{1} to {2}".format(project, user,
                                         reassign_editor_form.cleaned_data['editor']))
+    elif 'embargo_files' in request.POST:
+        embargo_files_days_form = forms.EmbargoFilesDaysForm(data=request.POST)
+        if settings.SYSTEM_MAINTENANCE_NO_UPLOAD:
+            raise ServiceUnavailable()
+        elif embargo_files_days_form.is_valid():
+            embargo_days_from_form = embargo_files_days_form.cleaned_data['embargo_files_days']
+            if embargo_days_from_form == 0:
+                project.embargo_files_days = None
+            else:
+                project.embargo_files_days = embargo_days_from_form
+            project.save()
```

Comment:
Perhaps add a "Embargo set" message here? e.g. something like:

```
messages.success(request, f'An embargo was set for {project.embargo_files_days} day/s.')
```

Reply:
Ok, sure.

Reply:
I've added this message.